text/3-entrees/off-chain-workers/transactions.md

```diff
@@ -296,18 +301,17 @@ impl<T: Trait> support::unsigned::ValidateUnsigned for Module<T> {
 }
 ```
 
-We implement `ValidateUnsigned`, and the allowance logic is added inside the `validate_unsigned`
-function. We check if the call is to `Call::submit_number_unsigned` and returns `Ok()` if this is
-the case. Otherwise, `InvalidTransaction::Call`.
+By implementing `ValidateUnsigned`, and the allowance logic is added inside the `validate_unsigned`
+function. We can verify if the call is to `Call::submit_number_unsigned` and returns `Ok()` otherwise, `InvalidTransaction::Call`.
```

Comment:
```suggestion
By implementing `ValidateUnsigned`, the allowance logic is added inside the `validate_unsigned`
function. We verify that if the call is `Call::submit_number_unsigned` we return `Ok()`, otherwise `InvalidTransaction::Call`.
```